CONTRIBUTING.md

```diff
@@ -91,7 +91,7 @@ Some examples of valid locale tags:
 
 ### Default language for newly added pages
 
-The default language used for pages is English (US). Pages written in English are stored in the default `pages` directory (notice the absence of a specific language tag). Although not strictly required, if you'd like to add a new page in a different language, please consider creating the English page too.
```

Comment:
I'm not sure this change makes it clearer. The `used` is important, as it indicates that the language is used on a pages.